scripts/AI/Sensors/HearingSensor.gd

```diff
@@ -0,0 +1,19 @@
+extends Node2D
+class_name HearingSensor
+
+var eye_location: Vector2
+@onready var _enemy: Enemy = $".."
+
+signal sound_heared(source: Node2D, location: Vector2, intensity: float)
+
+func _ready():
+	if !GlobalSignals.sound_emitted.is_connected(_on_heard_sound):
+		GlobalSignals.sound_emitted.connect(_on_heard_sound)
```

Comment:
You can connect it in the GUI (right panel) and remove this part

![image](https://github.com/gabriel654165/Very-Good-Cop/assets/58398928/8332b451-6181-4517-b138-ca586ee0f0e1)

Reply:
I don't find how to connect the GlobalSignal from the GUI.
Do you know how to do it ?